### PR TITLE
Master refactor + 2021.3.5s

### PR DESCRIPTION
**Pull request by Galster-dev**
Things done:
- [x] Switch to reactor
- [x] Refactor the code
- [x] Update to 2021.3.5s
- [ ] Update docs

### CODE REVIEW

**Comment by MatuxGG** [comment]
Insane job man. Hopefully, it will be added soon 😮

File: src/CrowdedMod/Patches/CreateGameOptionsPatches.cs [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+            {
+                if (__instance.mode != SettingsMode.Host) return;
+                var offset = __instance.MaxPlayerButtons[1].transform.position.x - __instance.MaxPlayerButtons[0].transform.position.x;
+
+                #region MaxPlayers stuff
+                
+                List<SpriteRenderer> playerButtons = __instance.MaxPlayerButtons.ToList();
+                
+                SpriteRenderer plusButton = Object.Instantiate(playerButtons.Last(), playerButtons.Last().transform.parent);
+                plusButton.GetComponentInChildren<TextRenderer>().Text = "+";
```
Comment (Galster-dev) [inline src/CrowdedMod/Patches/CreateGameOptionsPatches.cs:26]: this TextRenderer does not instantiate causing cosmetic bugs 
image
It's all working, just this

**Comment by przebor** [comment]
I experienced weird issues with second page of meetinghud. Tested on latest Impostor, maybe that's the issue.
TO-DO:

- [ ] create game button fix
- [ ] fix for second page and more (might require custom impostor)

**Comment by Galster-dev** [comment]
works fine for me (unpatched a bit old Impostor server)
image

**Pull request merged**